tests/store/api/test_base.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for the BaseHandle class."""
+from typing import Type
+
+import pytest
+from dataclasses import astuple
+
+from alchy import Query, ModelBase
+
+from cg.store.api.base import BaseHandler
+
+
+@pytest.mark.parametrize("table", astuple(BaseHandler()))
+def test__get_query(base_store, table: Type[ModelBase]):
+    """Tests the _get_query function for all attributes of BaseHandler ie tables in the database."""
+    assert isinstance(base_store._get_query(table=table), Query)
```

Comment:
> Looks fantastic ⭐ . I think we should add some unit tests for the refactored "get query" functions. However, it would be good to merge this asap so we can use this and extend it in the other PRs so we can also do a simple unit test PR after this one



@henrikstranneheim Actually all the queries are tested here 😊

Reply:
How did I miss that one 🥇